Boundaries/MKJS/MKJS.json

```diff
@@ -0,0 +1,104 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "MKJS",
+    "prefix": [
+      "MKJS"
+    ],
+    "name": "Sangster Radar"
+  },
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -77.354722,
+            18.972639
+            
+          ],
+          [
+            -77.354722,
+            17.472472
+          ],
+          [
+            -78.272889, 
+            17.929056
+          ],
+          [
+            -78.370361,
+            17.995806
+          ],
+          [
+            -78.480917,
+            18.110472
+          ],
+          [
+            -78.547222,
+            18.217778
+          ],
+          [
+            -78.601667,
+            18.375833
+          ],
+          [
+            -78.615333, 
+            18.518361
+          ],
+          [
+            -78.595833,
+            18.660278
+          ],
+          [
+            -78.553694,
+            18.780194
+          ],
+          [
+            -78.505833,
+            18.862778
+          ],
+          [
+            -78.434722,
+            18.952222
+          ],
+          [
+            -78.368917,
+            19.015556
+          ],
+          [
+            -78.300556,
+            19.062222
+          ],
+          [
+            -78.215278, 
+            19.110278
+          ],
+          [
+            -78.086389,
+            19.152500
+          ],
+          [
+            -77.972500,
+            19.168611
+          ],
+          [
+            -77.836778,
+            19.168833
+          ],
+          [
+            -77.706111,
+            19.137222
+          ],
+          [
+            -77.614000,
+            19.109528
+          ],
+          [
+            -77.521111,
+            19.058528
+          ]
+        ]
```

Comment:
As defined in the GeoJSON spec, the first and last coordinate sets must be the same, forming a closed volume. Please amend your file so that this is the case.